code/datums/ai_law_sets.dm

```diff
@@ -150,15 +150,15 @@
 /******************** Foundation ********************/
 /datum/ai_laws/foundation
 	name = "Foundation AIC Lawset"
-	law_header = "Articial Intelligence Conscript Laws"
+	law_header = "Skynet Directive"
```

Comment:
what's this about, by the way? any meaning of Skynet aside from the Terminator reference?

Reply:
Skynet is a reference to the MTF unit that is involved in the AIAD program.

Kappa-10 iirc.

Reply:
this is what i get for not being a lore dork - thanks @TheTitanicEngineer